app/assets/javascripts/component-directives/form_buttons.js

```diff
@@ -0,0 +1,8 @@
+ManageIQ.angularApplication.directive('formButtons', function() {
+  return {
+    restrict: 'E',
+    scope: false,
+    controller: 'buttonGroupController',
+    templateUrl: 'buttons.html'
+  };
+});
```

Comment:
The `buttonGroupController` implementation should probably live here too.. (but maybe only after we're using the component everywhere?)